### ISSUE TITLE
Crash IndexError

### ISSUE THREAD

**Issue by CaffeinatedTech** [vinifmor/fpakman#30]
Tried both AUR and PIP install version 0.4.1

I can see the tray icon, and I get a notification for available updates, but when I try to open the app it just crashes.

-- Edit --
I actually see a window with a "refreshing" bar for a second before it crashes.

```
$ fpakman
qt5ct: using qt5ct plugin
qt5ct: D-Bus system tray: no
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/fpakman/view/qt/thread.py", line 92, in run
    self.signal.emit(self.manager.read_installed())
  File "/usr/lib/python3.7/site-packages/fpakman/core/controller.py", line 169, in read_installed
    installed.extend(man.read_installed(disk_loader=disk_loader))
  File "/usr/lib/python3.7/site-packages/fpakman/core/snap/controller.py", line 77, in read_installed
    return [self.map_json(app_json, installed=True, disk_loader=disk_loader) for app_json in snap.read_installed()]
  File "/usr/lib/python3.7/site-packages/fpakman/core/snap/snap.py", line 84, in read_installed
    apps.append(app_str_to_json(app_str))
  File "/usr/lib/python3.7/site-packages/fpakman/core/snap/snap.py", line 43, in app_str_to_json
    'notes': app_data[5]
IndexError: list index out of range
Aborted (core dumped)
```
```
$ inxi -Fxxx
System:    Host: adam-pc Kernel: 5.2.1-1-MANJARO x86_64 bits: 64 compiler: gcc v: 9.1.0 Desktop: Cinnamon 4.2.2 
           wm: muffin dm: LightDM 1.30.0 Distro: Manjaro Linux 
Machine:   Type: Desktop Mobo: ASUSTeK model: PRIME X470-PRO v: Rev X.0x serial: <root required> 
           UEFI [Legacy]: American Megatrends v: 4024 date: 09/07/2018 
CPU:       Topology: 6-Core model: AMD Ryzen 5 2600X bits: 64 type: MT MCP arch: Zen+ rev: 2 L2 cache: 3072 KiB 
           flags: avx avx2 lm nx pae sse sse2 sse3 sse4_1 sse4_2 sse4a ssse3 svm bogomips: 86267 
           Speed: 2284 MHz min/max: 2200/3600 MHz boost: enabled Core speeds (MHz): 1: 1956 2: 1942 3: 3954 4: 4007 
           5: 1954 6: 2191 7: 2717 8: 2882 9: 2643 10: 2675 11: 2595 12: 2446 
Graphics:  Device-1: Advanced Micro Devices [AMD/ATI] Ellesmere [Radeon RX 470/480/570/570X/580/580X/590] 
           vendor: Micro-Star MSI driver: amdgpu v: kernel bus ID: 09:00.0 chip ID: 1002:67df 
           Display: x11 server: X.Org 1.20.5 driver: amdgpu unloaded: modesetting alternate: ati,fbdev,vesa 
           resolution: 3840x2160~60Hz 
           OpenGL: renderer: Radeon RX 580 Series (POLARIS10 DRM 3.32.0 5.2.1-1-MANJARO LLVM 8.0.0) 
           v: 4.5 Mesa 19.1.2 direct render: Yes 
Audio:     Device-1: AMD Ellesmere HDMI Audio [Radeon RX 470/480 / 570/580/590] vendor: Micro-Star MSI 
           driver: snd_hda_intel v: kernel bus ID: 09:00.1 chip ID: 1002:aaf0 
           Device-2: Advanced Micro Devices [AMD] Family 17h HD Audio vendor: ASUSTeK driver: snd_hda_intel 
           v: kernel bus ID: 0b:00.3 chip ID: 1022:1457 
           Device-3: Blue Microphones Yeti Stereo Microphone type: USB driver: hid-generic,snd-usb-audio,usbhid 
           bus ID: 5-2:2 chip ID: b58e:9e84 serial: REV8_2017/10/28_13435 
           Sound Server: ALSA v: k5.2.1-1-MANJARO 
Network:   Device-1: Intel I211 Gigabit Network vendor: ASUSTeK driver: igb v: 5.6.0-k port: e000 bus ID: 07:00.0 
           chip ID: 8086:1539 
           IF: enp7s0 state: up speed: 1000 Mbps duplex: full mac: 2c:fd:a1:bc:15:17 
Drives:    Local Storage: total: 1.14 TiB used: 970.41 GiB (83.3%) 
           ID-1: /dev/nvme0n1 vendor: Samsung model: SSD 960 EVO 250GB size: 232.89 GiB speed: 31.6 Gb/s lanes: 4 
           serial: S3ESNX0K365544J rev: 3B7QCXE7 scheme: MBR 
           ID-2: /dev/sda vendor: Samsung model: SSD 850 EVO 1TB size: 931.51 GiB speed: 6.0 Gb/s 
           serial: S3NZNF0JB07322P rev: 3B6Q scheme: GPT 
Partition: ID-1: / size: 211.28 GiB used: 195.18 GiB (92.4%) fs: ext4 dev: /dev/nvme0n1p1 
           ID-2: swap-1 size: 17.22 GiB used: 274.0 MiB (1.6%) fs: swap dev: /dev/nvme0n1p2 
Sensors:   System Temperatures: cpu: 50.0 C mobo: N/A gpu: amdgpu temp: 50 C 
           Fan Speeds (RPM): cpu: 0 gpu: amdgpu fan: 1097 
Info:      Processes: 400 Uptime: 12h 24m Memory: 15.66 GiB used: 6.39 GiB (40.8%) Init: systemd v: 242 Compilers: 
           gcc: 9.1.0 alt: 4.5 clang: 8.0.0 Shell: bash v: 5.0.7 running in: gnome-terminal inxi: 3.0.34 
```

```
$ flatpak list
Name                                                         Application ID                                    Version              Branch            Installation
Vocal                                                        com.github.needleandthread.vocal                  2.4.2                stable            system
Franz                                                        com.meetfranz.Franz                               5.1.0                stable            system
Endless Sky                                                  io.github.EndlessSky.endless-sky                  0.9.8                stable            system
Freedesktop.org Application Platform version 1.6             org.freedesktop.Platform                          1.6                  1.6               system
Freedesktop Platform                                         org.freedesktop.Platform                          18.08.34             18.08             system
FFmpeg extension                                             org.freedesktop.Platform.ffmpeg                                        1.6               system
html5-codecs                                                 org.freedesktop.Platform.html5-codecs                                  18.08             system
GNOME Application Platform version 3.30                      org.gnome.Platform                                                     3.30              system
Adwaita theme                                                org.kde.KStyle.Adwaita                                                 5.12              system
KDE Application Platform                                     org.kde.Platform                                                       5.12              system
Kdenlive                                                     org.kde.kdenlive                                  19.04.3              stable            system
```

**Comment by vinicius-moreira-dafiti** [comment]
I could identify the error. Thank you for your feedback. I will be publishing a fix release as soon as possible. For now, disable Snap usage to not get this error. You can do it via an environment variable (**FPAKMAN_SNAP=0**) or a parameter (**fpakman --snap=0**).

**Comment by vinifmor** [comment]
I could identify the error. Thank you for your feedback. I will be publishing a fix release as soon as possible. For now, disable Snap usage to not get this error. You can do it via an environment variable (FPAKMAN_SNAP=0) or a parameter (fpakman --snap=0).

**Comment by vinifmor** [comment]
I published a possible fix to the staging branch (**0.4.2.RC-1**). Could you, please, test **fpakman-staging** and tell me if the error is still happening ? If it is still happening, paste here the result of **snap list** command so I can figure out what's happening.

**Comment by CaffeinatedTech** [comment]
The 0.4.2 build in staging works well.  I used it to remove and install both snaps and flatpaks.  Very handy app.  Thank you.
